### PR TITLE
updated policy info

### PR DESCRIPTION
**Pull request by JoshHilliker**
updated with policy info for General & Memory Optimized.

### CODE REVIEW

**Review by kevinbleckmann — APPROVED** [review]
LGTM

**Review by kevinbleckmann — COMMENTED** [review]
LTGM

File: policies.md [hunk 1]
```diff
@@ -10,12 +10,15 @@
...
+## AWS Elasticache Redis Policy
 
-Description: **Add the description of the policy. This will be removed !**
+Description: The configured "instance_types" should be Intel Xeon 3rd Generation(code-named Ice Lake). At the time of publication of this policy, Elasticache does not enable any Ice Lake or Intel 4th gen Xeon Scalable processors (code named Sapphire Rapids).
 
-Resource type:  **Add the resource type of the policy. This will be removed !**
+Resource type: autoscaling
 
-Parameter:  **Add the Parameter. This will be removed !**
+Parameter: instance_types
```
Comment (lucasmelogithub) [inline policies.md:19]: Replace 'instance_types' with 'node_type'

File: policies.md [hunk 2]
```diff
@@ -10,12 +10,15 @@
...
 This file documents the HashiCorp Sentinel policies that apply to this module
 
-## Policy 1
+## AWS Elasticache Redis Policy
 
-Description: **Add the description of the policy. This will be removed !**
+Description: The configured "instance_types" should be Intel Xeon 3rd Generation(code-named Ice Lake). At the time of publication of this policy, Elasticache does not enable any Ice Lake or Intel 4th gen Xeon Scalable processors (code named Sapphire Rapids).
 
-Resource type:  **Add the resource type of the policy. This will be removed !**
+Resource type: autoscaling
```
Comment (lucasmelogithub) [inline policies.md:17]: Replace 'autoscaling' with 'aws_elasticache_replication_group'

File: policies.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img src="https://github.com/intel/terraform-intel-aws-elasticache-redis/blob/main/images/logo-classicblue-800px.png?raw=true" alt="Intel Logo" width="250"/>
+  <img src="./images/logo-classicblue-800px.png" alt="Intel Logo" width="250"/>
```
Comment (lucasmelogithub) [inline policies.md:2]: Update to full URL

Comment (JoshHilliker) [reply policies.md:2]: fixed..

File: policies.md (hunk 2 above)
Comment (JoshHilliker) [reply policies.md:17]: fixed

File: policies.md (hunk 1 above)
Comment (JoshHilliker) [reply policies.md:19]: fixed

File: policies.md [hunk 4]
```diff
@@ -10,12 +10,15 @@
 
 This file documents the HashiCorp Sentinel policies that apply to this module
 
-## Policy 1
+## AWS Elasticache Redis Policy
 
-Description: **Add the description of the policy. This will be removed !**
+Description: The configured "instance_types" should be Intel Xeon 3rd Generation(code-named Ice Lake). At the time of publication of this policy, Elasticache does not enable any Ice Lake or Intel 4th gen Xeon Scalable processors (code named Sapphire Rapids).
```
[the remark this answers is not in the document]
Comment (JoshHilliker) [reply policies.md:15]: fixed

**Review by lucasmelogithub — APPROVED** [review]
Approving

**Pull request merged**